inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/constant/IcebergConstant.java

```diff
@@ -35,8 +35,7 @@ public class IcebergConstant {
     public static final String START_SNAPSHOT_ID = "start-snapshot-id";
     public static final String STREAMING = "streaming";
     public static final String STARTING_STRATEGY_KEY = "starting-strategy";
-
-    public static final String APPEND_MODE_KEY = "appendMode";
+    public static final String APPEND_MODE_KEY = "upsert-enabled";
```

Comment:
Append and upsert have different meanings and naming may not seem easy to understand

Reply:
fixed, thx